src/renderer/pages/Settings/Networks/Networks.tsx

```diff
@@ -195,10 +190,14 @@ export const Networks = () => {
             <InactiveNetwork networkItem={network}>
               <NetworkSelector
                 networkItem={network}
+                openEditRpcModal={() => editCustomRpcModel.events.processStarted(true)}
```

Comment:
Seems awkward that we need to pass a boolean flag to the event that claims process start but requires something besides void 🤔
`editCustomRpcModel.events.processStarted(true)` 

> Summary of 'Clean code' by Robert C. Martin
> Function rules
> 6. Don't use flag arguments. Split method into several independent methods that can be called from the client without the flag.

Reply:
Agreed, but this is the event that manages whether the modal is open or not. I had no better idea, maybe we could simply have a better name.

Reply:
I just saw some of your code that was merged recently. I would have thought this isn't ok.. but if it is, then I can do such a construct here too and have a nice event name to call.

```ts
sample({
  clock: flowStarted,
  fn: () => Step.INIT,
  target: stepChanged,
});
```

Reply:
changed it now, with things I originally thought were not ok, with sample and `fn: () => false`